hack/local-up-kubeedge.sh

```diff
@@ -35,6 +34,7 @@ function check_prerequisites {
 # spin up cluster with kind command
 function kind_up_cluster {
   echo "Running kind: [kind create cluster ${CLUSTER_CONTEXT}]"
+  kind delete cluster ${CLUSTER_CONTEXT}
```

Comment:
If it doesn't run kind in host, will this leadto exit?

Reply:
yes , it is a bug. i will delete this line